tests/test_fall_back.py

```diff
@@ -0,0 +1,40 @@
+# from typing import Dict, Tuple
```

Comment:
For now, we decided to merge the PR without the test but will create an issue to address it in the future.